### PR TITLE
DOCS-9357 behavioral headers

### PR DESCRIPTION
**Pull request by sleslie23**

### CODE REVIEW

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 1]
```diff
@@ -34,16 +30,48 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
```
Comment (iamerico) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:37]: We could change it for:
For example, suppose that an API specification contained multiple examples within a response:

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 2]
```diff
@@ -34,16 +30,48 @@ Examples:
...
+
+----
+/examples:
```
Comment (iamerico) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:40]: There is an error in the example in the ISSUE, the **application/json** node is missing. It should be:
```/examples:
  get:
    responses:
      200:
        body:
          application/json:
            examples:
              one:
                prop1: something
                prop2: 30
              two:
                prop1: another
                prop2: 31
              three:
                prop1: anything
                prop2: 33

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 3]
```diff
@@ -34,16 +30,48 @@ Examples:
...
+              prop2: 30
+            two:
+              prop1: another
+              prop2: 31
+            three:
+              prop1: anything
+              prop2: 33
+----
+
+Here, the possible values of the header would be `one`, `two`, and `three`. Sending the value `one` as part of the request would return the properties `something` and `30`. Sending a value other than `one`, `two`, and `three` would return no results.
```
Comment (iamerico) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:57]: Change the word properties to payload.
Also, change: would return no results. to would return an error message.

**Review by iamerico — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 1 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:37]: Changed.

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 2 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:40]: Updated the example.

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 3 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:57]: Updated.

**Review by iamerico — APPROVED** [review]
LGTM

**Review by rciccone91 — APPROVED** [review]

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 4]
```diff
@@ -34,16 +30,51 @@ Examples:
...
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
+
+For example, you can have an API specification that contains multiple examples within a response as shown in the following example:
```
Comment (rddheath) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:39]: "For example, ... multiple examples ... as shown in the following example" -> "This example shows three examples for the status code 200:"

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 5]
```diff
@@ -34,16 +30,51 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
```
Comment (rddheath) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:37]: "within a response:" -> "within a response."

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 6]
```diff
@@ -34,16 +30,51 @@ Examples:
...
+                prop2: 30
+              two:
+                prop1: another
+                prop2: 31
+              three:
+                prop1: anything
+                prop2: 33
+----
+
+Here, the possible values of the header would be `one`, `two`, and `three`. Sending the value `one` as part of the request would return the payload `something` and `30`. Sending a value other than `one`, `two`, and `three` would return an error message.
```
Comment (rddheath) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:60]: "Sending a value other than `one`, `two`, and `three`" -> "Sending a value other than `one`, `two`, or `three`"

File: modules/ROOT/pages/apid-behavioral-headers.adoc [hunk 7]
```diff
@@ -34,16 +30,51 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
```
Comment (rddheath) [inline modules/ROOT/pages/apid-behavioral-headers.adoc:35]: "for the same status code" -> "for a single status code"

**Review by rddheath — APPROVED** [review]

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 4 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:39]: Changed!

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 5 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:37]: Fixed

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 6 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:60]: Thanks! Fixed!

File: modules/ROOT/pages/apid-behavioral-headers.adoc (hunk 7 above)
Comment (sleslie23) [reply modules/ROOT/pages/apid-behavioral-headers.adoc:35]: Changed.!

**Review by rddheath — APPROVED** [review]

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**